content/summits/sparse/meeting1.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
There's a similar problem with the rendering of the "summary" of this page in parent sparse summits page: it looks like the entire doc is being picked up as raw text and dropped into the summary blurb: see [here](https://deploy-preview-280--scientific-python-org.netlify.app/summits/sparse/)

Reply:
See above.